services/user.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = "",
+        last_name: str = "",
+) -> None:
+
+    get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name,
+    )
+
+
+def get_user(user_id: int) -> User:
+
+    return get_user_model().objects.get(pk=user_id)
+
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = None,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+) -> None:
+
+    user = get_user(user_id)
+
+    if username:
+        user.username = username
+    if password:
+        user.set_password(password)
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
```

Comment:
Consider wrapping the 'update_user' function in a transaction using '@transaction.atomic' to ensure that all updates are applied atomically. This will prevent partial updates in case of an error.